Examples/CMakeLists.txt

```diff
@@ -1 +1,73 @@
-add_subdirectory (CppClient)
+set (
+  CARLA_EXAMPLE_CPP_CLIENT_SOURCE_PATH
+  ${CARLA_WORKSPACE_PATH}/Examples/CppClient
+)
+
+set (
+  CARLA_EXAMPLE_CPP_DEPENDENCIES
+  carla-client
+  rpc
+  png_static
+  zlibstatic
+  Boost::asio
+  Boost::python
+  Boost::geometry
+  Boost::gil
+  RecastNavigation::Recast
+  RecastNavigation::Detour
+  RecastNavigation::DetourCrowd
+)
+
+if (BUILD_EXAMPLES)
```

Comment:
This if can be removed: we check for BUILD_EXAMPLES being on in the top CMakeLists.txt file, in which case all subprojects in Examples/ will be skipped.

Reply:
updated